shop/shop-controller/src/main/java/shop/woowasap/shop/controller/ProductController.java

```diff
@@ -0,0 +1,24 @@
+package shop.woowasap.shop.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import shop.woowasap.shop.dto.UpdateProductRequest;
+import shop.woowasap.shop.service.ProductService;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/v1/products")
+public class ProductController {
+
+    private final ProductService productService;
+
+    @PutMapping("/{product-id}")
+    public ResponseEntity<Void> updateProduct(@PathVariable("product-id") Long productId, UpdateProductRequest request) {
```

Comment:
NIL: 이건 그냥 궁금한 건데, `product-id` 로 작성하신 이유가 있는지 궁금해요! 아래와 같이 작성할 수도 있을 것 같아서 궁금해졌어요!

```suggestion
    @PutMapping("/{productId}")
    public ResponseEntity<Void> updateProduct(@PathVariable Long productId, UpdateProductRequest request) {
```

Reply:
저희 노션에 있는 API 명세대로 작성했습니다!